### PR TITLE
vtr-optimized: remove conda-forge

### PR DESCRIPTION
**Pull request by acomodi**
Fixes the vtr-optimized package build

Signed-off-by: Alessandro Comodi <acomodi@antmicro.com>

### CODE REVIEW

**Review by mithro — COMMENTED** [review]
Does `requirements:` support specifying things in `pip`? It doesn't look like it, but it would be worth checking;
 - https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#requirements-section
 - https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#create-env-file-manually

**Comment by acomodi** [comment]
@mithro I have looked a bit at the docs and it does indeed seem that is not possible to install custom pip packages through the `requirements` except by getting them through conda.

**Comment by mithro** [comment]
> @mithro I have looked a bit at the docs and it does indeed seem that is not possible to install custom pip packages through the `requirements` except by getting them through conda.

We should probably log a bug about that on upstream conda. Go ahead and merge otherwise.

**Review by mithro — APPROVED** [review]
LGTM

File: pnr/vtr-optimized/build.sh [hunk 1]
```diff
@@ -36,6 +36,10 @@ PACKAGE[architecture]=symbiflow-xc7a50t_test-latest
 
 BUILD_ROOT=$PWD
 
+# python-constraint requires to use the conda-forge channel.
+# Given that it is only used to build this package, install it via the GH repository.
+python3 -m pip install git+https://github.com/python-constraint/python-constraint.git
```
Comment (PiotrZierhoffer) [inline pnr/vtr-optimized/build.sh:41]: we tend to use `$PYTHON`, can you please change it?

**Review by PiotrZierhoffer — APPROVED** [review]

**Pull request merged**